tools/physiofit/physiofit.xml

```diff
@@ -1,278 +1,98 @@
-                                <add value='{'/>
-                                <add value='}'/>
-                            </valid>
-                        </sanitizer>
-                    </param>
-                </when>
-            </conditional>
-
-            <conditional name='cm_select'>
-                <param name='cm' type='select' label='Flag to add bounds on metabolite concentrations'>
-                    <option value='No'>No</option>
-                    <option value='Yes'>Yes</option>
-                </param>
-                <when value='Yes'>
-                    <param name='conc_met_bounds' type='text' value='()' label='Bounds on metabolite concentrations. Bounds should be given in tuple format (see docs for more information)'/>
-
-                </when>
-            </conditional>
-
-            <conditional name='fm_select'>
-                <param name='fm' type='select' label='Select yes to add bounds on metabolite fluxes'>
-                    <option value='No'>No</option>
-                    <option value='Yes'>Yes</option>
-                </param>
-                <when value='Yes'>
-                    <param name='flux_met_bounds' type='text' value='()' label='Bounds on metabolite fluxes. Bounds should be given in tuple format (see docs for more information)' />
-                </when>
-            </conditional>
-
-            <conditional name='cb_select'>
-                <param name='cb' type='select' label='Select yes to add bounds on biomass concentrations'>
-                    <option value='No'>No</option>
-                    <option value='Yes'>Yes</option>
-                </param>
-                <when value='Yes'>
-                    <param name='conc_biom_bounds' type='text' value='()' label='Bounds on biomass concentrations. Bounds should be given in tuple format (see docs for more information)' />
-                </when>
-            </conditional>
-
-            <conditional name='fb_select'>
-                <param name='fb' type='select' label='Select yes to add bounds on biomass fluxes'>
-                    <option value='No'>No</option>
-                    <option value='Yes'>Yes</option>
-                </param>
-                <when value='Yes'>
-                    <param name='flux_biom_bounds' type='text' value='()' label='Bounds on biomass fluxes. Bounds should be given in tuple format (see docs for more information)' />
-                </when>
-            </conditional>
-
-            <param name='verbose' type='boolean' argument='--verbose' truevalue='-v' falsevalue='' label='Flag to get debug information'/>
-
-        </section>
-
+        <param name='datafile' type='data' format='tabular, txt' multiple='true' label='Upload data file containing biomass and concentration values'/>
+        <param name='configfile' type='data' format='yaml' multiple='true' label='Upload yaml configuration file containing run parameters'/>
```

Comment:
Can't you provide the different `<param>` to the user and template the configfile?

Reply:
No depending on the selected models and such the config file changes and as such it is simpler to have the user fill in the different run parameters beforehand